app/src/shared/components/ModalDialog/index.jsx

```diff
@@ -11,12 +11,15 @@ import styles from './modalDialog.pcss'
 
 export type Props = {
     onClose: () => void,
-    children: Node,
     className?: string,
     backdropClassName?: string,
 }
 
-const ModalDialog = ({ onClose, children, className, backdropClassName }: Props) => (
+export type InternalProps = Props & {
```

Comment:
FYI, this is the issue I raised up on Slack, i.e. `{ key?: type } & { key: type }`. Flow gives a very sketchy error message for it that does not help figuring out wtf.